Shapes.java

```diff
@@ -1,25 +1,77 @@
-import java.util.*;
-
-public class Shapes{
+import java.util.Scanner;
+public class Shapes {
   public static void main(String[] args) {
+    areaRectangle();
+    areaTriangle();
+    areaCircle();
+    areaTrapezoid();
+
+  }//end main method
+
+  public static void areaRectangle(){
+    Scanner input = new Scanner(System.in);
+    int length = 0;
+    int width = 0;
+
+    System.out.print("The width of the rectangle is: ");
+    width = input.nextInt();
+
+    System.out.print("The length of the rectangle is: ");
+    length = input.nextInt();
+
+    int areaRectangle = length * width;
+    System.out.println("The area of the rectangle is: " + areaRectangle);
+
+  }//end areaRectangle
+
+  public static void areaTriangle(){
```

Comment:
Can you figure out a way to change the return type to something other than `void`?